pkg/frontend/kubeactions/kubeactions.go

```diff
@@ -160,6 +163,63 @@ func (ka *kubeactions) List(ctx context.Context, oc *api.OpenShiftCluster, kind,
 	return ul.MarshalJSON()
 }
 
+func (ka *kubeactions) applyOne(ctx context.Context, dyn dynamic.Interface, grs []*restmapper.APIGroupResources, un *unstructured.Unstructured) error {
+	if strings.EqualFold(un.GetKind(), "secret") { // TODO do we need this?
+		return fmt.Errorf("access to secrets is forbidden")
+	}
+	gvrs := ka.findGVR(grs, un.GetKind())
+	if len(gvrs) == 0 {
+		return fmt.Errorf("Kind %s not found", un.GetKind())
+	}
+	if len(gvrs) > 1 {
+		return fmt.Errorf("Kind %s did not uniquely match one GRV", un.GetKind())
```

Comment:
internal errors start lower case so that they can be composed.  GRV is misspelled.  Reuse the error message from Get().